package.json

```diff
@@ -27,10 +27,10 @@
     "@babel/preset-react": "^7.10.4",
     "@babel/preset-typescript": "^7.18.6",
     "@changesets/cli": "2.24.2",
-    "@ima/cli": "18.0.0-rc.7",
-    "@ima/core": "18.0.0-rc.7",
+    "@ima/cli": "18.0.0-rc.8",
+    "@ima/core": "18.0.0-rc.8",
     "@ima/dev-utils": "^18.0.0-rc.7",
-    "@ima/plugin-cli": "^18.0.0-rc.2",
+    "@ima/plugin-cli": "^18.0.0-rc.3",
     "@ima/server": "18.0.0-rc.7",
     "@types/jest": "^26.0.24",
     "@types/node": "^18.7.6",
```

Comment:
I've tried this, but figured out that esm in jest is not as simple as renaming these files. 

CLI plugin is currently transpiled from TS to CJS, so it would not work too. 

I would probably leave this to some other PR and not complicate this any more. Also CLI plugins needs to be updated to work with ESM (which will probably happen)